inc/woocommerce.php

```diff
@@ -0,0 +1,239 @@
+<?php
+/**
+ * WooCommerce Compatibility File
+ *
+ * @link https://woocommerce.com/
+ *
+ * @package _s
+ */
+
+/**
+ * WooCommerce setup function.
+ *
+ * See: https://docs.woocommerce.com/document/third-party-custom-theme-compatibility/
+ *
+ * @return void
+ */
+function _s_woocommerce_setup() {
+	add_theme_support( 'woocommerce' );
+	add_theme_support( 'wc-product-gallery-zoom' );
+	add_theme_support( 'wc-product-gallery-lightbox' );
+	add_theme_support( 'wc-product-gallery-slider' );
+}
+
```

Comment:
In other places in the code we do not have the extra line break between the function and hook.